src/components/AdvancedFacetViewModal/index.js

```diff
@@ -61,7 +66,40 @@ class AdvancedFacetViewModalContent extends React.Component {
             {...props}
             InputComponent={CustomFilterInput}
             sqon={localSqon}
-            onSqonChange={({ sqon }) => effects.setAdvancedFacetSqon(sqon)}
+            onSqonChange={({ sqon, field, value }) => effects.setAdvancedFacetSqon(sqon)}
+            onSqonSubmit={({ sqon }) => {
+              trackFileRepoInteraction({
+                category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
+                action: 'View Results',
+                label: sqon,
+              });
+            }}
+            onFilterChange={val => {
+              trackFileRepoInteraction({
+                category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
+                action: TRACKING_EVENTS.actions.filter + ' - Search',
+                ...(val && { label: val }),
+              });
+            }}
+            onTermSelected={({ field, value, active }) => {
+              if (active) {
+                trackFileRepoInteraction({
+                  category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
+                  action: TRACKING_EVENTS.actions.filter + ' Selected',
+                  label: {
+                    type: 'filter',
+                    value,
+                    field,
+                  },
+                });
+              }
+            }}
+            onClear={() => {
+              trackFileRepoInteraction({
+                category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
+                action: TRACKING_EVENTS.actions.query.clear,
+              });
+            }}
```

Comment:
for these callbacks, also compose in the callbacks from props. So the follow for example:
```
onSqonSubmit={({ sqon, ...rest }) => {
  trackFileRepoInteraction({
    category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
    action: 'View Results',
    label: sqon,
  });
  if(props.onSqonSubmit) {
    props.onSqonSubmit({sqon, ...rest})
  }
}}
```

or, consider moving this to the higher level